examples/some-verifier/frontend/src/@types/global.d.ts

```diff
@@ -0,0 +1,9 @@
+import { Platform, Issuer } from '../lib/types';
+
+declare global {
+  declare const config: {
+    discordClientId: string;
+    telegramBotName: string;
```

Comment:
Didn't we agree that it would make more sense to move the network declaration from each issuer to the global config, or did I misunderstand?

Reply:
We did. I changed it now.